source/schema.txt

```diff
@@ -338,3 +338,424 @@ To delete a location filter from the map:
+   sampling, see :ref:`Sampling <sampling>`.
+
+Once your schema has been analyzed, export your schema:
+
+1. In the top menu bar, click :guilabel:`Collection`.
+
+#. From the dropdown, click :guilabel:`Share Schema as JSON`.
+
+.. figure:: /images/compass/share-schema-json.png
+   :alt: Image showing Collection dropdown
+
+Your schema is copied to your clipboard as a |json| object.
+
+Schema Object Properties
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Schema objects have properties ``count`` and ``fields``:
+
+- ``count`` is an integer that represents the number of documents 
+  :ref:`sampled <sampling>` from the collection to generate the schema.
+
+- ``fields`` is an array of ``n`` metadata objects that represent each 
+  field in the documents analyzed during sampling. Each element in the 
+  ``fields`` array contains:
+
+.. list-table::
+  :header-rows: 1
+  :widths: 25 35 40
+
+  * - Field name
+    - Data type
+    - Description
+
+  * - ``name``
+    - String
+    - Name of ``fields[n]``, e.g. ``_id``.
+
+  * - ``path``
+    - String
+    - Path to ``fields[n]`` within a document.
+    
+  * - ``count``
+    - Integer
+    - The number of documents that ``fields[n]`` appears on.
+
+  * - ``types``
+    - Array
+    - List of ``n`` metadata objects that represent each data type that 
+      appears in ``fields[n]``.
+
+  * - ``types[n].name``
+    - String
+    - Data type of ``types[n]``.
+  
+  * - ``types[n].bsonType``
+    - String
+    - |bson| type of ``types[n]``.
+
+  * - ``types[n].path``
+    - String
+    - Path to ``name`` of ``fields[n]`` within a document.
+
+  * - ``types[n].count``
+    - Integer
+    - Number of times ``types[n]`` appears in ``fields[n]``.
+  
+  * - ``types[n].values``
+    - Array
+    - List of the actual sampled values that 
+      appear in ``fields[n]`` and match ``types[n]``.
+
+  * - ``types[n].total_count``
+    - Integer
+    - Number of items nested in ``field[n]``. 0 when 
+      ``fields[n]`` is not a nested value.
+  
+  * - ``types[n].probability``
+    - Number
+    - Probability that the value of ``field[n]`` is ``types[n]`` in a 
+      random document.
+
+  * - ``types[n].unique``
+    - Integer
+    - Number of unique values of ``types[n]`` that appear in 
+      ``fields[n]``.
+
+  * - ``types[n].has_duplicates``
+    - Boolean
+    - ``true`` if a single value of ``types[n]`` appears multiple times 
+      in ``fields[n]``. Otherwise ``false``.
+
+  * - ``types[n].lengths``
+    - Array
+    - If ``types[n]`` is an array, an array of integers representing 
+      the lengths of arrays found in ``fields[n]``. Not present for 
+      other data types.
+
+  * - ``types[n].average_length``
+    - Number
+    - If ``types[n]`` is an array, the average length of arrays in 
+      ``fields[n]`` across sampled documents. Not present for 
+      other data types.
+
+  * - ``total_count``
+    - Integer
+    - Number of documents sampled from the collection.
```

Comment:
That contain `fields[n]`? or total number overall?

Reply:
Total overall - I just double checked on a sample airbnb collection. I'll call out for tech review just in case.